framework/include/base/ExecuteMooseObjectWarehouse.h

```diff
@@ -91,131 +96,123 @@ class ExecuteMooseObjectWarehouse : public MooseObjectWarehouse<T>
-ExecuteMooseObjectWarehouse<T>::operator[](ExecFlagType exec_flag)
+    ExecuteMooseObjectWarehouse<T>::
+    operator[](ExecFlagType exec_flag)
 {
   // Use find to avoid accidental insertion
-  typename std::map<ExecFlagType, MooseObjectWarehouse<T> >::iterator iter = _execute_objects.find(exec_flag);
+  typename std::map<ExecFlagType, MooseObjectWarehouse<T>>::iterator iter = _execute_objects.find(exec_flag);
 
   if (iter == _execute_objects.end())
     mooseError("Unable to locate the desired execute flag, the global list of execute parameters is likely out-of-date.");
 
   return iter->second;
 }
 
-
-template<typename T>
+template <typename T>
 void
-ExecuteMooseObjectWarehouse<T>::updateActive(THREAD_ID tid/* = 0 */)
+ExecuteMooseObjectWarehouse<T>::updateActive(THREAD_ID tid /* = 0 */)
 {
   // Update all objects active list
   MooseObjectWarehouse<T>::updateActive(tid);
 
   // Update the execute flag lists of objects
-  typename std::map<ExecFlagType, MooseObjectWarehouse<T> >::iterator iter;
+  typename std::map<ExecFlagType, MooseObjectWarehouse<T>>::iterator iter;
   for (iter = _execute_objects.begin(); iter != _execute_objects.end(); ++iter)
     iter->second.updateActive(tid);
 }
 
-
-template<typename T>
+template <typename T>
 void
-ExecuteMooseObjectWarehouse<T>::jacobianSetup(THREAD_ID tid/* = 0*/) const
+ExecuteMooseObjectWarehouse<T>::jacobianSetup(THREAD_ID tid /* = 0*/) const
 {
   checkThreadID(tid);
-  typename std::map<ExecFlagType, MooseObjectWarehouse<T> >::const_iterator iter = _execute_objects.find(EXEC_NONLINEAR);
+  typename std::map<ExecFlagType, MooseObjectWarehouse<T>>::const_iterator iter = _execute_objects.find(EXEC_NONLINEAR);
   if (iter != _execute_objects.end())
     iter->second.jacobianSetup(tid);
 }
 
-
-template<typename T>
+template <typename T>
 void
-ExecuteMooseObjectWarehouse<T>::residualSetup(THREAD_ID tid/* = 0*/) const
+ExecuteMooseObjectWarehouse<T>::residualSetup(THREAD_ID tid /* = 0*/) const
 {
   checkThreadID(tid);
-  typename std::map<ExecFlagType, MooseObjectWarehouse<T> >::const_iterator iter = _execute_objects.find(EXEC_LINEAR);
+  typename std::map<ExecFlagType, MooseObjectWarehouse<T>>::const_iterator iter = _execute_objects.find(EXEC_LINEAR);
   if (iter != _execute_objects.end())
     iter->second.residualSetup(tid);
 }
 
-
-template<typename T>
+template <typename T>
 void
-ExecuteMooseObjectWarehouse<T>::setup(ExecFlagType exec_flag, THREAD_ID tid/* = 0*/) const
+ExecuteMooseObjectWarehouse<T>::setup(ExecFlagType exec_flag, THREAD_ID tid /* = 0*/) const
 {
   checkThreadID(tid);
   switch (exec_flag)
   {
-  case EXEC_INITIAL:
-    initialSetup(tid);
-    break;
-  case EXEC_TIMESTEP_BEGIN:
-    timestepSetup(tid);
-    break;
-  case EXEC_SUBDOMAIN:
-    subdomainSetup(tid);
-    break;
-  case EXEC_NONLINEAR:
-    jacobianSetup(tid);
-    break;
-  case EXEC_LINEAR:
-    residualSetup(tid);
-    break;
-  default:
-    break;
+    case EXEC_INITIAL:
```

Comment:
This is fine with me.  We need to make sure to update our `.emacs` to do the right thing... ;-)